tests/phpunit/CRM/Utils/GeocodeTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Class CRM_Utils_GeocodeTest
+ * @group headless
+ */
+class CRM_Utils_GeocodeTest extends CiviUnitTestCase {
+
+  public function setUp() {
+    parent::setUp();
+  }
+
+  public function tearDown() {
+    parent::tearDown();
+  }
+
+  public function testStateProvinceFormat() {
+    $params = array('state_province_id' => 1022, 'country' => 'U.S.A');
+    $formatted = CRM_Utils_Geocode_Google::format($params);
+    $this->assertTrue($formatted);
+    $this->assertEquals('46.729553', $params['geo_code_1']);
```

Comment:
Perhaps this should be using `assertApproxEquals()`? It feels weird to assign such a precise coordinate to a landmass as large as the United States.